todo-list-service/src/main/java/org/example/todo_list_service/Todos.java

```diff
@@ -0,0 +1,21 @@
+
+package org.example.todo_list_service;
+
+public class Todos {
```

Comment:
Let's rename this class as `Todo` (in singular) because it abstracts a single to-do item